### ISSUE TITLE
NTR: Modifier: Joints: predominantly large affected (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7968]
New term request Modifier: Joints: predominantly large affected (segal_130201180935): Includes hips, shoulders, knees; elbows are medium

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000110
- label: Large joint predominance
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Refers to a phenotypic abnormality that mainly affect large joints.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7968
- synonyms:
